.github/workflows/workflow-release-services.yaml

```diff
@@ -56,10 +56,7 @@ jobs:
 
       - name: Perform maven snapshot release
         if: inputs.snapshot-release == true && inputs.maven-release == true
-        run: >
-          mvn -B -ntp deploy
-          --activate-profile release
-          -Darguments="-DskipTests"
+        run: mvn --batch-mode --no-transfer-progress deploy --activate-profiles release -DskipTests
```

Comment:
Das mit `clean`, `--active-profiles` und `--no-transfer-progress` sollte man dann noch an https://github.com/it-at-m/digiwf-core/pull/685/files#diff-26bf6fb02f259d518d4e460d774d013e2995e62bd4f71e7219331e99fe16c088R51 angleichen

Reply:
Und dann sind die Commands gleich außer man will das `release`-Profile nur für einen richtigen Release, dann müsste man das aber verlegen.

Reply:
Stimmt, dann müssen wir hier gar nicht unterscheiden zwischen Snapshot und richtigem Release. Ich passe das gleich noch an, dass wir da nur einen Step haben 👍🏻